src/js/covidMap.js

```diff
@@ -583,6 +580,13 @@ fluid.defaults("fluid.covidMap.map", {
             priority: "last",
             func: "{query}.accept",
             args: [0]
+        },
+        "showHospitalPanel": {
```

Comment:
In the modern style this is done by a direct relay from "isHospitalShowing" onto the path of the class as in the following test: https://github.com/fluid-project/infusion/blob/main/tests/framework-tests/core/js/FluidIoCViewTests.js#L418-L423

Reply:
The logic of this block is: if `isHopitalShowing === true`, remove the class; if it's false, add the class. I tried by added `!` before the class name as: `segs: ["dom", "hospitalPanel", "class", "!{that}.options.styles.hideHospitalPanel"]`. It doesn't work.

Is there a way to express the "not" operation in "segs"?

Reply:
To negate, you add the entry in 

    func: x => !x

Reply:
Right! It works and ready for another look. Thanks.